app/src/main/res/layout/include_new_browser_tab.xml

```diff
@@ -54,12 +54,13 @@
 
     <include
         layout="@layout/include_dax_dialog_cta"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
         android:layout_marginTop="40dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_max="600dp"/>
```

Comment:
you are using this value 4 times, maybe it makes sense to extract as a dimension?

Reply:
Good point, I know they are the same number, but it's just because the same size worked well to constraint each dialog for the different screens. But there's no indication that if at some point we need to tweak one of those dialogs, the rest should change too. So they are not officially related between them.